pom.xml

```diff
@@ -652,7 +652,7 @@
             </systemProperties>
             <redirectTestOutputToFile>${test.redirectToFile}</redirectTestOutputToFile>
             <useFile>${test.redirectToFile}</useFile>
-            <argLine>${argLine} -Xmx1g -XX:MaxPermSize=256m</argLine>
+            <argLine>${argLine} -Xmx4g -XX:MaxPermSize=1024m</argLine>
```

Comment:
Please don't change these; the travis machines are not very large and this might cause memory pressure. If the extra memory is really needed we should investigate why and, if it's really needed, change the travis config also to use larger machines if possible.